server/controllers/controller_helpers/metrics.js

```diff
@@ -0,0 +1,115 @@
+import _ from 'lodash';
+
+import responsesService from '../../db/services/responses';
+import campaignsService from '../../db/services/campaigns';
+
+function splitResponses(responseString) {
+  return responseString.split(',');
+}
+
+export function mapCallIds(callObjArray) {
+  return callObjArray.map(call => call.attributes.id);
+}
+
+export function checkCampaignNotDraft(campaignId, res) {
```

Comment:
Should we just default everything to zero for draft campaigns to remain consistent across the view Campaign views?